types/relay-test-utils/lib/RelayMockPayloadGenerator.d.ts

```diff
@@ -1,27 +1,21 @@
-import {
-    Variables,
-    NormalizationField,
-    NormalizationOperation,
-    NormalizationSelection,
-    NormalizationLinkedField,
-    NormalizationScalarField,
-    OperationDescriptor,
-    GraphQLResponse,
-    NormalizationSplitOperation,
-} from 'relay-runtime';
+import { GraphQLSingularResponse, OperationDescriptor } from 'relay-runtime';
 
-interface MockResolverContext {
-    parentType?: string | undefined;
-    name?: string | undefined;
-    alias?: string | undefined;
-    path?: ReadonlyArray<string> | undefined;
-    args?: { [key: string]: any } | undefined;
+export interface MockResolverContext {
+    parentType?: string | null | undefined;
+    name?: string | null | undefined;
+    alias?: string | null | undefined;
+    path?: ReadonlyArray<string> | null | undefined;
+    args?: { [key: string]: any } | null | undefined;
```

Comment:
I think: `args: Record<string, unknown> | null | undefined;`

Reply:
ya this was all existing, should have double checked them! 🤝 thanks will update